iphone/Classes/CalendarModule.m

```diff
@@ -294,6 +294,66 @@ -(NSNumber*) calendarAuthorization
     return [NSNumber numberWithInteger:result];
 }
 
+-(TiCalendarCalendar*) createCalendar:(id)args
+{
+    ENSURE_SINGLE_ARG(args, NSDictionary);
+    
+    NSString* name = nil;
+    ENSURE_ARG_FOR_KEY(name, args, @"name", NSString);
```

Comment:
Didn't even know that macro, nice catch! So it grabs the key, ensures the type, and assigns it? Nice.